2218/0/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3'
+services:
+  redmine:
+    build: .
+    links:
+      - db
+    volumes:
+      - .:/app
+    command: [ "sh", "-c", "rm -f /app/tmp/pids/server.pid \
```

Comment:
can you explain what is it mean?
why do you need all these lines in command: ?
and what will happen every time you start your container?
Also, what happens to the data if I delete your container?